src/components/footer.js

```diff
@@ -67,21 +67,77 @@ const CustomWrap = styled.div`
   text-align: center;
 `;
 
-const Footer = () => (
-  <FooterWrap>
-    <div className={wrap} />
-    <CustomWrap />
-    <div className={wrap}>
-      <Dropdown>
-        <div className={dropdownContent}>
-          <Element>1</Element>
-          <Element>2</Element>
-          <Element>3</Element>
+class Footer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentLanguage: 'en',
+    };
+  }
+
+  changeLanguage = (lng) => {
+    i18n.changeLanguage(lng);
+    this.setState({ currentLanguage: lng });
+  };
+
+  chooseFlag = (lng) => {
+    if (lng === 'en') {
+      return (
+        <img
+          className="icon"
+          src="https://cdn.pg.edu.pl/ekontakt-updated-theme/images/language/en_GB.png"
+          alt="English (United Kingdom)"
+          title="English (United Kingdom)"
+        />
+      );
+    } if (lng === 'pl') {
+      return (
+        <img
+          className="icon"
+          src="https://cdn.pg.edu.pl/ekontakt-updated-theme/images/language/pl_PL.png"
+          alt="polski (Polska)"
+          title="polski (Polska)"
+        />
+      );
+    }
+    return null;
+  }
+
+  render() {
+    const { currentLanguage } = this.state;
+    return (
+      <FooterWrap>
+        <div className={wrap} />
+        <CustomWrap />
+        <div className={wrap}>
+          <Dropdown>
+            <div className={dropdownContent}>
+              <Element onClick={() => this.changeLanguage('pl')}>
```

Comment:
Czy to będzie wyświetlać oba języki niezależnie od tego jaki jest wyświetlony? To jest oczywiście okej i w niczym nie pzreszkadza po prostu odpalisz funkcje zmiany języka na już wybrany np, ale potencjalnie ładniejszym rozwiązaniem byłoby mieć języki dostępne POZA wybranym. To jest tylko sugestia nie musisz tego koniecznie zmieniać.

Masz za to kilka elementów które różniąs sie językami i to potencjalnie jest kandydat na użycie array.map i zrobienie arraya z dostępnymi językami jako jakiś const który importujesz żeby mieć dostępne języki w jednym osobnym miejscu. Łatwiej dodać języki wtedy.